accounts/models.py

```diff
@@ -87,8 +87,8 @@ def has_module_perms(self, app_label):
 		# Simplest possible answer: Yes, always
 		return True
 
-	@property

```

Comment:
Was getting an error when trying to modify is_staff so removed it.
https://stackoverflow.com/questions/62504785/typeerror-user-got-an-unexpected-keyword-argument-is-staff-when-using-custo

Reply:
If the @property method is removed , I was able to modify it.

Reply:
I was able to modify is_staff value after removing the @property method.